sdk/python/kfp/components/placeholders.py

```diff
@@ -270,10 +292,12 @@ def __str__(self) -> str:
 
 _CONTAINER_PLACEHOLDERS = (IfPresentPlaceholder, ConcatPlaceholder)
 PRIMITIVE_INPUT_PLACEHOLDERS = (InputValuePlaceholder, InputPathPlaceholder,
-                                InputUriPlaceholder, InputMetadataPlaceholder)
+                                InputUriPlaceholder, InputMetadataPlaceholder,
+                                InputListOfArtifactsPlaceholder)
 PRIMITIVE_OUTPUT_PLACEHOLDERS = (OutputParameterPlaceholder,
                                  OutputPathPlaceholder, OutputUriPlaceholder,
                                  OutputMetadataPlaceholder)
+# TODO: output lists of placeholders -- add to PRIMITIVE_OUTPUT_PLACEHOLDERS
```

Comment:
Added TODOs in two places where we'll need to add logic to handle output placeholders once we enable their usage